hibernate-core/src/main/java/org/hibernate/cfg/EJB3DTDEntityResolver.java

```diff
@@ -41,13 +41,27 @@ public boolean isResolved() {
 	public InputSource resolveEntity(String publicId, String systemId) {
 		LOG.tracev( "Resolving XML entity {0} : {1}", publicId, systemId );
 		if ( systemId != null ) {
-			if ( systemId.endsWith( "orm_2_1.xsd" ) ) {
+			if ( systemId.endsWith( "orm_3_0.xsd" ) ) {
+				InputStream dtdStream = getStreamFromClasspath( "orm_3_0.xsd" );
+				final InputSource source = buildInputSource( publicId, systemId, dtdStream, false );
+				if ( source != null ) {
+					return source;
+				}
+			}
+			else if ( systemId.endsWith( "orm_2_1.xsd" ) ) {
 				InputStream dtdStream = getStreamFromClasspath( "orm_2_1.xsd" );
 				final InputSource source = buildInputSource( publicId, systemId, dtdStream, false );
 				if ( source != null ) {
 					return source;
 				}
 			}
+			else if ( systemId.endsWith( "orm_2_2.xsd" ) ) {
+				InputStream dtdStream = getStreamFromClasspath( "orm_2_2.xsd" );
```

Comment:
Was something missing here for JPA 2.2 ?

Reply:
I didn't see an entry for http://www.oracle.com/webfolder/technetwork/jsc/xml/ns/persistence/orm_2_2.xsd which I think was added for https://jakarta.ee/specifications/persistence/2.2.  Note that the Jakarta EE 8 Platform TCK did not update to the 2.2 schemas but some applications may of.

Reply:
ok. Would you say we need to track it as a separate bugfix?